src/App.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/extensions */
```

Comment:
- [ ] As stated in the previous review this kindly remove this eslint disable rule.